scheduling/airflow.py

```diff
@@ -1,10 +1,18 @@
+import argparse
+
 from pyspark import SparkConf, SparkContext
 from pyspark.sql import SQLContext
 from python_etl import main
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='')
```

Comment:
I don't think we need this method at all. An ETL job could simply be invoked with e.g.:
```
python -m python_etl.example_job --start-date X --end-date Y
```